ray-operator/test/e2e/support.go

```diff
@@ -175,3 +178,54 @@ func jobSubmitterPodTemplateApplyConfiguration() *corev1ac.PodTemplateSpecApplyC
 						corev1.ResourceMemory: resource.MustParse("500Mi"),
 					}))))
 }
+
+func IsEnvVarExisted(env string, pod *corev1.Pod) bool {
```

Comment:
Use `utils.EnvVarExists` instead, in case the KubeRay operator packages have already been imported.